tests/profiler/core/test_utils.py

```diff
@@ -138,7 +142,8 @@ def test_timeline_merge_by_step(out_dir, trace_location):
 
     assert total_events == num_trace_events
 
-
+@pytest.mark.skipif(version.parse(tf.__version__) > version.parse("2.10"),
+                    reason="tf 2.11 merged trace.json file to xplane.pb file.")
```

Comment:
why do you need to skip all tests related to merging timelines? only one file is not available, rests still should work